### ISSUE TITLE
Allow different payment amounts for different participants

### ISSUE THREAD

**Issue by jhamrick** [Dallinger/Dallinger#409]
Currently, all participants are paid at the same rate (specified in `base_payment`). You can modify this, of course, by giving people bonuses, but it might also be nice to just be able to specify different base payments for different groups of participants. For example, if I have some participants doing my main task, and another group doing a validation task, then I have to make sure those tasks take approximately the same amount of time otherwise one group is going to get paid much more (or less) than what I would like. It would be nice to specify one base pay for the main group, and a different base pay for the other group. As I mentioned, this could be done with bonuses, but that seems like a rather complex solution when what I want to be able to do is relatively simple.

**Comment by suchow** [comment]
Any ideas on how to accomplish this? I think it would require setting up multiple HIT types in an experiment. Maybe each class of node could be assigned its own recruiter.

**Comment by thomasmorgan** [comment]
Yes, I think having a single base_payment is baked into MTurk so if you want to pay people differentially you can either set up different hits (hard to do with dallinger) or use bonusing (easier - perhaps even easy - to do with dallinger).

My concern with doing payment through bonuses is that participants will see a very low base payment advertised, however, I think Nori has used the flexible bonusing approach before with success, so I guess its ok
